snapshots/terraform/stack/snapshot_generator/service.tf

```diff
@@ -28,9 +28,10 @@ module "snapshot_generator" {
   cluster_arn  = var.cluster_arn
 
   security_group_ids = [
-    var.elastic_cloud_vpce_sg_id,
+    # TODO: Do we need this egress security group?
```

Comment:
Easy to test by manually removing this in the console and seeing what happens I guess?

Reply:
Yeah, I just didn't want to do it right now – I've broken enough stuff!